### PR TITLE
Instructions how to integrate with API ML in Zowe in Docker

### PR DESCRIPTION
**Pull request by plavjanik**
Documentation on how to integrate with Zowe in Docker (provided and described by @VitekVlcek-Broadcom at https://github.com/zowe/zowe-dockerfiles/tree/master/dockerfiles/zowe-1.5.0) has been added.

It shows how to make Zowe in Docker to trust the sample service and how the Zowe in Docker can connect to the service running on the Docker host even if they share the same hostname by using `host.docker.internal` as the hostname.

The "localhost" sample certificate has been updated to include `host.docker.internal` in the SAN extension so the certificate validation succeeds.

Signed-off-by: Petr Plavjanik <plavjanik@gmail.com>

### CODE REVIEW

File: zowe-rest-api-sample-spring/docs/zowe-integrate-with-apiml.md [hunk 1]
```diff
@@ -74,6 +74,59 @@ It has two sections:
 
 The description of the properties are in the Zowe documentation.
 
+## Integration with the Zowe in Docker on Your Computer
```
Comment (dkelosky) [inline zowe-rest-api-sample-spring/docs/zowe-integrate-with-apiml.md]: Probably should describe that it's integration of the sample API service with Zowe API ML in Docker (and so throughout).

I think "Zowe" is broad and might confuse zlux.

File: zowe-rest-api-sample-spring/docs/zowe-integrate-with-apiml.md [hunk 2]
```diff
@@ -74,6 +74,59 @@ It has two sections:
...
+The certificates that were set up during this process are stored in `certs` directory. We will refer to this directory as `$CERTS`.
+The `server.p12` contains the server certificate and Zowe APIML truststore.
+
+In the directory with the sample service, we issue the following command to get the local CA certificate trusted by the Zowe in Docker:
+
+```bash
+keytool -exportcert -keystore config/local/truststore.p12 -storepass password -alias localca -rfc > $CERTS/localca.cer
+```
+
+Then the Zowe in Docker can be started. You have to restart it, if it is running.
```
Comment (dkelosky) [inline zowe-rest-api-sample-spring/docs/zowe-integrate-with-apiml.md]: I'm not clear how we can start after the export, is some other step missing?

File: zowe-rest-api-sample-spring/docs/zowe-integrate-with-apiml.md [hunk 3]
```diff
@@ -74,6 +74,59 @@ It has two sections:
...
+
+If the Zowe server certificate at `$CERTS/server.p12` is signed by a public CA such as DigiCert, then you do not need to do anything since the `config/localhost/truststore.p12` already contains them.
+
+If the root CA is not a public one then you need to import to your truststore:
```
Comment (dkelosky) [inline zowe-rest-api-sample-spring/docs/zowe-integrate-with-apiml.md]: Which keystore does "your" refer to?

File: zowe-rest-api-sample-spring/docs/zowe-integrate-with-apiml.md [hunk 4]
```diff
@@ -74,6 +74,59 @@ It has two sections:
...
+
+* To list the certificate chain:
+
+    ```bash
+    keytool -list -keystore $CERTS/server.p12 -storepass password --rfc
+    ```
+
+* The last certificate should be the root CA. Save the last certificate to `rootca.pem` and then import it to the truststore:
```
Comment (dkelosky) [inline zowe-rest-api-sample-spring/docs/zowe-integrate-with-apiml.md]: Similar to above, perhaps qualify which truststore is meant by "the truststore"

**Comment by plavjanik** [comment]
@dkelosky Thank you for your comments. They are really helpful and I will address them soon.

**Pull request closed without merging**